src/Gateway/Blocks/PayplugOney4x.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Payplug\PayplugWoocommerce\Gateway\Blocks;
+
+class PayplugOney4x extends PayplugOney {
+
+	/**
+	 * Payment method name/id/slug.
+	 *
+	 * @var string
+	 */
+	protected $name = "oney_x4_with_fees";
+
+	protected $icon = 'x4_with_fees.svg';
+
+
+	public function oney_enabled() {
```

Comment:
this should be name `get_payment_method_data` shouldn't it?

Reply:
it is being called on the as `return $this->oney_enabled()` on the `get_payment_method_data`